Pipfile

```diff
@@ -91,7 +91,7 @@ service_factory = ">=0.1.5"
 pytest-watch = "*"
 rope = "*"
 "autopep8" = "<=1.4.0"
-pre-commit= "==1.11.1"
+pre-commit= "==1.15.1"
```

Comment:
What's the reason for the upgrade?

Reply:
There was an issue with the "large files check" pre-commit hook, which was calling `git --lfs` checks even for projects that don't use it. Apparently with old versions of git this was okay, but in later versions it was changed to throw an error, which then triggered a failed check. Sorry, I actually thought this change had already been merged, I wonder if others updated locally?